install_config/aggregate_logging.adoc

```diff
@@ -398,28 +387,34 @@ endif::openshift-enterprise[]
 
 == Post-deployment Configuration
 
+[[aggregated-elasticsearch]]
 === Elasticsearch
 
-In any production employment, Elasticsearch should be deployed with a
-cluster size of at least three for resiliency to node failures. Each instance
-requires individual storage, but an OpenShift deployment can only provide
-volumes shared by all its pods, so the Elasticsearch cluster cannot be
-implemented with a single deployment. The EFK deployer instead creates one
-deployment per instance. To view all current deployments of Elasticsearch:
+In a production environment, Elasticsearch should be deployed with a cluster
+size of at least three hosts to account for resiliency. Each Elasticsearch
+replica requires its own storage, but an OpenShift deployment configuration
+shares storage volumes between all its pods. So, when scaled up, the EFK
+deployer ensures each replica of Elasticsearch has its own deployment
+configuration.
+
+*Viewing all Elasticsearch Deployments*
+
+To view all current Elasticsearch deployments:
 
 ====
 ----
 $ oc get dc --selector logging-infra=elasticsearch
 ----
 ====
 
+[[aggregated-logging-persistent-storage]]
 *Persistent Elasticsearch Storage*
 
-The deployer creates an ephemeral deployment in which all of a pod's
-data is lost upon restart. For production usage, persistent storage is
-required. You can use the `oc volume` command to add a created volume
-to each deployment. The following example specifies a volume for an
-Elasticsearch instance (using a
+The deployer creates an ephemeral deployment in which all of a pod's data is
+lost upon restart. For production usage, persistent storage is required by
+adding a created volume to each deployment. 
```

Comment:
"required by" feels a little awkward. How about:

Production usage requires adding a persistent storage volume to each Elasticsearch deployment configuration.